modules/installation-guide/examples/system-variables.adoc

```diff
@@ -83,39 +83,39 @@
- `+CHE_INFRA_KUBERNETES_PVC_PRECREATE__SUBPATHS+`,"`+true+`","Defines whether to run a job that creates workspace's subpath directories in persistent volume for the 'common' strategy before launching a workspace. Necessary in some versions of OpenShift/Kubernetes as workspace subpath volume mounts are created with root permissions, and thus cannot be modified by workspaces running as a user (presents an error importing projects into a workspace in {prod-short}). The default is 'true', but should be set to false if the version of Openshift/Kubernetes creates subdirectories with user permissions. Relevant issue: \https://github.com/kubernetes/kubernetes/issues/41638 Note that this property has effect only if the 'common' PVC strategy used." 
+ `+CHE_INFRA_KUBERNETES_PVC_STRATEGY+`,"`+common+`","Defined which strategy will be used while choosing PVC for workspaces. Supported strategies: - 'common'        All workspaces in the same {platforms-namespace} will reuse the same PVC.        Name of PVC may be configured with 'che.infra.kubernetes.pvc.name'.        Existing PVC will be used or new one will be created if it doesn't exist. - 'unique'        Separate PVC for each workspace's volume will be used.        Name of PVC is evaluated as '{che.infra.kubernetes.pvc.name} + '-' + {generated_8_chars}'.        Existing PVC will be used or a new one will be created if it doesn't exist. - 'per-workspace'        Separate PVC for each workspace will be used.        Name of PVC is evaluated as '{che.infra.kubernetes.pvc.name} + '-' + {WORKSPACE_ID}'.        Existing PVC will be used or a new one will be created if it doesn't exist." 
+ `+CHE_INFRA_KUBERNETES_PVC_PRECREATE__SUBPATHS+`,"`+true+`","Defines whether to run a job that creates workspace's subpath directories in persistent volume for the 'common' strategy before launching a workspace. Necessary in some versions of {orch-name} as workspace subpath volume mounts are created with root permissions, and thus cannot be modified by workspaces running as a user (presents an error importing projects into a workspace in {prod-short}). The default is 'true', but should be set to false if the version of {orch-name} creates subdirectories with user permissions. Relevant issue: \https://github.com/kubernetes/kubernetes/issues/41638 Note that this property has effect only if the 'common' PVC strategy used." 
```

Comment:
> Necessary in some versions of {orch-name} as workspace subpath volume mounts are created with root permissions

Are we sure it is also the case on OpenShift ? cc @skabashnyuk

Reply:
I believe - yes.

Reply:
OK, thanks.